shared/src/message.rs

```diff
@@ -93,3 +93,21 @@ impl<'de> Deserialize<'de> for Message {
         })
     }
 }
+
+#[cfg(feature = "arb")]
+impl quickcheck::Arbitrary for Message {
+    fn arbitrary(g: &mut quickcheck::Gen) -> Self {
+        Self {
+            to: Address::new_id(u64::arbitrary(g)),
```

Comment:
We have an arbitrary implementation for `Address`. Let's use it:

```suggestion
            to: Address::arbitrary(g),
```

Reply:
:+1: